### PR TITLE
Dynamically get the binary file

### PR DESCRIPTION
**Pull request by Tieske**
replace #470 to be able to run CI without secrets

### CODE REVIEW

**Review by hutchic — APPROVED** [review]

File: ubuntu/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN set -ex; \
     if [ "$ASSET" = "ce" ] ; then \
         apt-get install -y curl && \
         curl -fL \
-		https://download.konghq.com/gateway-${KONG_VERSION%%.*}.x-ubuntu-xenial/pool/all/k/kong/kong_${KONG_VERSION}_$(dpkg --print-architecture).deb -o /tmp/kong.deb \
```
[the remark this answers is not in the document]
Comment (gszr) [reply ubuntu/Dockerfile:18]: also, curious as for why this is needed, as the `FROM` hardcodes the base image

[the remark this answers is not in the document]
Comment (gszr) [reply ubuntu/Dockerfile:18]: also, curious why this is needed, as the FROM hardcodes a base image

[the remark this answers is not in the document]
Comment (Tieske) [reply ubuntu/Dockerfile:18]: this was already merged through #470

[the remark this answers is not in the document]
Comment (Tieske) [reply ubuntu/Dockerfile:18]: > also, curious why this is needed, as the FROM hardcodes a base image
in the future we only need to update the FROM clause. Whereas so far we had to update both.

[the remark this answers is not in the document]
Comment (Tieske) [reply ubuntu/Dockerfile:18]: > let's add back the indentation

good catch, see #473

**Pull request closed without merging**